### ISSUE TITLE
[Bug]: Thumbnails/Screenshots of games not visible on page 20

### ISSUE THREAD

**Issue by rahul-p19** [kunjgit/GameZone#3091]
### Give a brief about the bug ✍️

On going to page 20 of the games, the thumbnail is not visible for many games, and an alt text is displayed.

### What is the expected behavior? 🤔

Proper thumbnail containing a screenshot of the game should be displayed.

### Provide step by step information reproduce the bug 📄

Currently the src attribute in the <img> tag is set to ./assets/images. I will add a suitable screenshots in the images folder and then add the appropriate src attribute.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

**Comment by rahul-p19** [comment]
@kunjgit Could you add GSSOC and Level labels to this issue. Thanks in advance.

**Comment by rahul-p19** [comment]
Please assign me this issue under GSSOC 24

**Comment by kunjgit** [comment]
Hey @rahul-p19 ! Thank you so much for your raising the issue💗 
 It’s all yours, you can come anytime again and make some contributions! 🚀 
 Alone, we can do little, but together we can do so much! 😇
